timely_beliefs/tests/test_belief_utils.py

```diff
@@ -31,3 +33,55 @@ def test_propagate_multi_sourced_deterministic_beliefs():
             == pd.Timestamp("2000-01-01 01:00:00+00:00")
         ].droplevel("belief_time"),
     )
+
+
+@pytest.mark.parametrize(
```

Comment:
Please include to begin with some regular case. I feel you only tested the special DST transition cases.